app/src/components/Item/AddItemForm.js

```diff
@@ -22,7 +22,12 @@ const AddItemForm = ({ currentContainerId, handleFormSubmit }) => {
   }
 
   const handleSubmit = (e) => {
-    handleFormSubmit(data);
+    //e.preventDefault();      can't add or it cannot redirect when click on submit btn
+    if(data.item_name===""){
```

Comment:
I believe there are other fields need to validate, you probably want to have a form validation method to wrap them.